### ISSUE TITLE
Another Word option may prevent embedded content from being displayed

### ISSUE THREAD

**Issue by nikola-nignite** [MicrosoftDocs/OfficeDocs-Support#1790]
For me, issue was that option **Show field codes instead of their values** option was ticked. When ticked, it was displaying placeholder text (e.g. { EMBED blablabla })

![image](https://user-images.githubusercontent.com/80263149/131624158-b23bc15f-fb64-4d2f-ab9e-6dda072b2fe5.png)

[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 46468fe2-ff0d-44ae-de1d-f7b411df3d36
* Version Independent ID: b63823d9-b8f7-ca2f-197f-16d1efe8372e
* Content: [A picture or an object may not appear in a Word 2010 or Word 2007 document - Office](https://docs.microsoft.com/en-us/office/troubleshoot/word/object-not-appear-in-word-2010)
* Content Source: [Office/Client/word/object-not-appear-in-word-2010.md](https://github.com/MicrosoftDocs/OfficeDocs-Support/blob/public/Office/Client/word/object-not-appear-in-word-2010.md)
* Product: **office-perpetual-itpro**
* GitHub Login: @helenclu
* Microsoft Alias: **luche**

**Comment by MaratMussabekov** [comment]
Hello @nikola-nignite ,
I am trying to reproduce the issue, but I am not able to.
Could you please advise, how did you embed the picture so it is not shown when **Show field codes instead of their values** option is enabled?
Thank you

**Comment by nikola-nignite** [comment]
I had a picture embedded in the document.
After some time picture stopped showing up. I did not know why. I don't think I messed around with any options. Probably the option was disabled when the document was created.

Anyhow, for a while I used document without seeing the picture. But when it did stop showing up, I found that **Show field codes instead of their values** was enabled. It sounded like it might be causing the issue. I disabled it and picture showed up. I also had some other fields in the document. Not sure now if they were some calculated fields, or references... They also started showing up values instead of field codes.
